site/content/en/docs/Reference/API/generatorOptions.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "generatorOptions"
+linkTitle: "generatorOptions"
+type: docs
+weight: 8
+description: >
+    Control behavior of [ConfigMap]() and
+    [Secret]() generators.
+---
+
+
+
+Additionally, generatorOptions can be set on a per resource level within each
+generator. For details on per-resource generatorOptions usage see
+[field-name-configMapGenerator]() and See [field-name-secretGenerator]().
```

Comment:
Should we remove these links that reference the same page?

Reply:
I made them refer to correct links.
https://github.com/kubernetes-sigs/kustomize/pull/5313/commits/ae041b5c54832ed26117bec8f29b38aab31b35fb